content/en/docs/chart_template_guide/function_list.md

```diff
@@ -903,9 +903,10 @@ the template engine if there is a problem.
 Returns a copy of the input string, replacing matches of the Regexp with the
 replacement string replacement. Inside string replacement, $ signs are
 interpreted as in Expand, so for instance $1 represents the text of the first
-submatch
+submatch. The first argument is `<pattern>`, second is `<input>`, and third is
+`<replacement>`.
```

Comment:
I think your PR will have a better chance of merging if you limit it to these two lines and the change on line 922.  Try just the english version.